src/controllers/language-listener/language-listener.js

```diff
@@ -0,0 +1,23 @@
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+export class LanguageListenerController {
+
+	constructor(host) {
+		(this.host = host).addController(this);
+		this._localeSettings = getDocumentLocaleSettings();
+		this._handleLangUpdate = () => this.host.requestUpdate();
+		this.language = this._localeSettings.language;
+	}
+
+	hostConnected() {
+		this._localeSettings.addChangeListener(this._handleLangUpdate);
+	}
+
+	hostDisconnected() {
+		this._localeSettings.removeChangeListener(this._handleLangUpdate);
+	}
+
+	hostUpdate() {
+		this.language = this._localeSettings.language;
```

Comment:
Couldn't this be set inside `_handleLangUpdate` and then there'd be no need for `hostUpdate`? I've only seen `hostUpdate` used to poke at the host's DOM before it's updated.